src/services/emrCluster/connections.ts

```diff
@@ -66,8 +72,34 @@ export default ({
     }
   }
 
+  /**
+   * Find IAM Roles
+   */
+  const iamRoles: {
+    name: string
+    data: { [property: string]: any[] }
+  } = data.find(({ name }) => name === services.iamRole)
+
+  if (iamRoles?.data?.[region]) {
+    const iamRolesInRegion: RawAwsIamRole[] = iamRoles.data[region].filter(
+      ({ RoleName }: RawAwsIamRole) =>
+        RoleName === ServiceRole || RoleName === AutoScalingRole
```

Comment:
interesting it uses roleName and not roleArn 🤔

Reply:
Yes, I know. AWS SDK and its edge cases.